app/src/main/java/pl/example/android/garbageapp/ui/ActivitySectorBlue.java

```diff
@@ -22,6 +28,11 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.rcSectorTerms.setLayoutManager(verticalLinearLayoutManager);
         binding.rcSectorTerms.setAdapter(sectorTermsAdapter);
 
+        DetailViewModelFactory factory = InjectorUtils.provideDetailViewModelFactory(this.getApplicationContext(), SectorType.BLUE);
+        mViewModel = ViewModelProviders.of(this, factory).get(DetailActivityViewModel.class);
```

Comment:
General comment regarding onCreate() methods in these 3 Activity classes: they are pretty much the same. I would extract ie. BaseActivitySector class which would override onCreate() and would contain the common code. In this PR you could start from extracting just mViewModel construction to the super onCreate method.

In the BaseActivitySector you can declare mViewModel variable as a field and then in onCreate() you can put all the following code:

`DetailViewModelFactory factory = InjectorUtils.provideDetailViewModelFactory(this.getApplicationContext(), sectorType());
mViewModel = ViewModelProviders.of(currentSector(), factory).get(DetailActivityViewModel.class);`

Extract methods ie. 
`protected Activity currentSector()`

and

`protected String sectorType()`

Next, override these methods in an appropriate Activity class.